java/refactoring.java/test/unit/src/org/netbeans/modules/refactoring/java/test/RefactoringTestBase.java

```diff
@@ -76,15 +78,46 @@ public class RefactoringTestBase extends NbTestCase {
 
     public RefactoringTestBase(String name) {
         super(name);
-        sourcelevel = "1.6";
+        sourcelevel = "17";//"1.6";//??
```

Comment:
17 is probably not a good default. Lets not change this value in the base class.